cmd/skopeo/main.go

```diff
@@ -54,6 +56,12 @@ func createApp() *cli.App {
 		if c.GlobalBool("debug") {
 			logrus.SetLevel(logrus.DebugLevel)
 		}
+		if c.GlobalIsSet("cert-path") {
+			logrus.Warn("'--cert-path' is deprecated, please set this on the specific subcommand")
+		}
+		if c.GlobalIsSet("tls-verify") {
+			logrus.Warn("'--tls-verify' is deprecated, please set this on the specific subcommand")
```

Comment:
I would go with Fatal.  Allowing the user to specify them and expecting something to happen, is not a good solution.  Especially for a security one like tls-verify.